multi/stm32l4-multi/flash.c

```diff
@@ -232,6 +244,68 @@ size_t flash_writeData(uint32_t offset, const char *buff, size_t size)
 }
 
 
+static unsigned int _getOptions(void)
+{
+	return *(flash_common.flash + flash_optr);
+}
+
+
+static int _setOptions(unsigned int mask, unsigned int val)
+{
+	int err;
+	unsigned int t;
+
+	if ((err = _flash_wait()) < 0)
+		return err;
+
+	_flash_clearFlags();
+
+	t = *(flash_common.flash + flash_optr) & ~mask;
```

Comment:
You could use _getOptions() & ~mask here

Reply:
I am not convinced to that because the currently used idiom (at least the dominant one in multidrv) for register read-modify-write operations is `t = reg & ~mask; reg = t | val` where `reg` is a register direct access expression. If _getOptions() changes in the future then this line here will be invalid.